### DISCUSSION TITLE
Why the name `ReducerProtocol` instead of `Reducer` ?

### DISCUSSION THREAD

**Discussion by amine2233** [pointfreeco/swift-composable-architecture#1885] [Q&A] [answered]
Hello, 

I'm very interesting to know why you choose `ReducerProtocol` name for a protocol instead of using just `Reducer` because naming `ReducerProtocol` for a protocol is a pleonasme.

**Comment by davdroman** [comment, marked as answer]
It will be renamed `Reducer` [in 1.0](https://github.com/pointfreeco/swift-composable-architecture/discussions/1477), but for now it's `ReducerProtocol` to maintain backwards compatibility with the old-style non-protocol `Reducer` type.

**Comment by amine2233** [comment]
Nice think you for your response
